src/persistence/user/cms_user_storage_null.e

```diff
@@ -65,6 +65,11 @@ feature -- Change: user
 			a_user.set_id (1)
 		end
 
+	new_user_from_temporal_user (a_user: CMS_USER)
```

Comment:
In fact I think it's not needed at all, because the interface does not check the user has_id in the postcondition.